.travis.yml

```diff
@@ -28,6 +28,7 @@ install:
       conda install -q numpy pytz python-dateutil;
       PRE_WHEELS="https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com";
       pip install --pre --upgrade --timeout=60 -f $PRE_WHEELS pandas;
+      pip install -e 'git+https://github.com/GoogleCloudPlatform/google-cloud-python.git#egg=version_subpkg&subdirectory=bigquery';
```

Comment:
I've struggled with these from time to time. I _think_ the `egg=version_subpkg` is like `foo` and can be excluded. But if this works then great

Reply:
Probably. I don't really want to mess with it, though. I got it from this [answer on installing with pip from a git subdirectory](https://stackoverflow.com/a/19516714/101923).

Reply:
It works!